tests/fs/test_llseek.c

```diff
@@ -14,6 +14,15 @@ int main()
     var stream = FS.open('testfile', 'a');
     fd = FS.write(stream, new Uint8Array([99, 61, 51]) /* c=3 */, 0, 3);
 
+    // check invalid whence
+    var ex;
+    try {
+      FS.llseek(stream, 0, 99);
+    } catch(e) {
+      ex = e;
+    }
+    assert(ex instanceof FS.ErrnoError && ex.errno === ERRNO_CODES.EINVAL);
```

Comment:
Looks like it's this one causing the failures.

Reply:
I see, ok. I think it's good enough to just hardcode the error here, then.